components/payments/internal/connectors/plugins/public/mangopay/webhooks.go

```diff
@@ -116,12 +116,13 @@ func (p Plugin) createWebhooks(ctx context.Context, req models.CreateWebhooksReq
 
 	webhookURL := fmt.Sprintf("%s/api/payments/v3/connectors/webhooks/%s", stackPublicURL, req.ConnectorID)
 	for eventType, config := range webhookConfigs {
+		url := fmt.Sprintf("%s%s", webhookURL, config.urlPath)
```

Comment:
maybe use `url.JoinPath` since it is designed for this purpose

Reply:
Done thanks !